packages/v4/src/content/design-guidelines/components/form/forms.md

```diff
@@ -113,6 +113,41 @@ Form submit buttons should be disabled until a user starts filling out the form
 * For a submitted form, disable the submit button while a user's responses are processing to avoid a duplicate submission.
 * For a processing form, use feedback messages and progress indicators like spinners or progress bars if the processing time might exceed user expectations.
 
+## Form sections
+Form sections sort form fields into non-hierarchical groups based on common information type, step, or visual preference. PatternFly form sections support a header followed by a left-aligned list of unnested and uncollapsible form groups, so they create a relatively flat information hierarchy. Form sections are ideal for breaking lengthy forms into simple, evenly spaced segments with all information exposed. 
```

Comment:
I feel like including this information here complicates things "PatternFly form sections support a header followed by a left-aligned list of unnested and uncollapsible form groups"

I don't think there's even a need to say unnested or uncollapsible here , because at it's core, form sections are just a way to divide up a form with a header and that's it. What can be underneath isn't as much of a concern compared to what it is